### ISSUE TITLE
Incorrect behaviour when trying to return to previous page

### ISSUE THREAD

**Issue by valumar** [PrivateBin/PrivateBin#503]
Incorrect behaviour when trying to return to previous page

## Steps to reproduce

1. Go to https://privatebin.net/?4f05a8046de22f26#8nUBijB2214JMGDwXTJ1n9EPvVzar2MCosN5rovNpyYj
2. Hit 'Raw Text'
3. Try to return to previous page by clicking to Back button in your browser
4. Nothing happens

### What happens

Nothing happens

### What should happen

Returning to previous page

## Basic information

**PrivateBin version:**
1.2.1 and 1.3

I can reproduce this issue on <https://privatebin.net>: Yes

**Comment by Haocen** [comment]
From my understanding this is intentional, as there are "burn after Reading" pastes that cannot be retrieve or displayed again, so browser actions, such as refresh and back are not recommended.

**Comment by elrido** [comment]
It is not really intentional, as I though we added extensive code to manipulate the browser history so it looks like the "Raw Text" leads to a new page, when it actually is just a manipulation of the current document.

It is correct that this only worked for the non-burn-after reading pastes, as the back button causes the browser to refresh the page instead. I tested a bit and it seems this is broken since 1.2, but used to still work in 1.1.

**Comment by Haocen** [comment]
What is the desired behavior, if I may ask?

Maybe we can setup a event listener to make this work, even for the burn after read paste:
https://developer.mozilla.org/en-US/docs/Web/API/Window/popstate_event

**Comment by Haocen** [comment]
Or simply open a new tab and write content into the new page? Would this be much easier?

**Comment by elrido** [comment]
Actually, you are right, @rugk had suggested in #270 to just offer a third tab next to preview to avoid all these history shenanigans and having to reload the paste, etc. That would also avoid the issue with the password in #94. And this was already on the project plan to be looked at for the next release.

**Comment by rugk** [comment]
I agree, this should be solved by https://github.com/PrivateBin/PrivateBin/issues/270. Leaving it open though as it is a seperate bug.
